### PR TITLE
Admin options

### PR DESCRIPTION
**Pull request by chandonn-zz**
Related to #4

### CODE REVIEW

File: admin/class-monk-admin.php [hunk 1]
```diff
@@ -124,45 +124,68 @@ public function monk_add_menu_page() {
...
 			array( $this, 'monk_language_select_render' ),
 			'general_options',
 			'monk_general_options'
 		);
+
+		register_setting( 'general_options', 'monk_active_languages' );
+		add_settings_field(
+			'monk_list',
+			__( 'Add new translation' ),
+			array( $this, 'monk_translation_list_render' ),
```
Comment (brenoalvs) [inline admin/class-monk-admin.php]: Change method name from `monk_translation_list_render` to `monk_active_languages_render` and adjust where is needed.

File: admin/class-monk-admin.php [hunk 2]
```diff
@@ -124,45 +124,68 @@ public function monk_add_menu_page() {
...
 			'monk_language_field',
 			__( 'Default language', 'monk' ),
 			array( $this, 'monk_language_select_render' ),
 			'general_options',
 			'monk_general_options'
 		);
+
+		register_setting( 'general_options', 'monk_active_languages' );
+		add_settings_field(
+			'monk_list',
```
Comment (brenoalvs) [inline admin/class-monk-admin.php]: Don't use generic names as `monk_list` use something like `monk_active_languages`  and adjust where is needed.

File: admin/class-monk-admin.php [hunk 3]
```diff
@@ -124,45 +124,68 @@ public function monk_add_menu_page() {
...
 	}
 
 	/**
-	 * Function to render a select field
+	 * Function to render the select field, callback for the monk_language_field element
 	 *
 	 * @since    1.0.0
-	*/
+	 */
 	public function monk_language_select_render() {
```
Comment (brenoalvs) [inline admin/class-monk-admin.php]: Change method name from `monk_language_select_render` to `monk_default_language_render` and adjust where is needed.

File: admin/partials/admin-monk-select-render.php [hunk 4]
```diff
@@ -10,19 +10,19 @@
  * @subpackage Monk/Admin/Partials
  */
 
-	$options = get_option( 'monk_language' );
+	$options = get_option( 'monk_default_language' );
 ?>
-	<select name="monk_language[language_select]">
-		<option value="pt-br"<?php selected( $options['language_select'], 'pt-br' ); ?>>
+	<select name="monk_default_language">
```
Comment (brenoalvs) [inline admin/partials/admin-monk-select-render.php]: Create an array called `$available_languages` and do a loop to render the language options.

File: admin/partials/admin-monk-translation-list-render.php [hunk 5]
```diff
@@ -0,0 +1,42 @@
+<?php
```
Comment (brenoalvs) [inline admin/partials/admin-monk-translation-list-render.php]: Fix some code standards and data escaping in the whole file.

File: admin/partials/admin-monk-translation-list-render.php [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+		foreach ($suported_languages as $lang_code => $lang_name) {
+
+			/**
+			 * Walks through each value in the active languages array creating the checkboxes
+			 * based on the values, making possible and more reliable updating that list 
+			*/
+
+			$id = strtolower( str_replace( '_', '', $lang_code) );
+	?>
+			<label for="monk-<?php echo $id; ?>">
```
Comment (brenoalvs) [inline admin/partials/admin-monk-translation-list-render.php]: Use checkboxes in WordPress style (copy the HTML structure):

screen shot 2016-10-02 at 10 28 21 pm
_print taked on General > Discussion menu_

File: admin/partials/admin-monk-translation-list-render.php [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+	<fieldset>
+	<?php
+		foreach ($suported_languages as $lang_code => $lang_name) {
+
+			/**
+			 * Walks through each value in the active languages array creating the checkboxes
+			 * based on the values, making possible and more reliable updating that list 
+			*/
+
+			$id = strtolower( str_replace( '_', '', $lang_code) );
```
Comment (brenoalvs) [inline admin/partials/admin-monk-translation-list-render.php]: Don't format strings using your own functions.
You can use a sanitize as `sanitize_title` here.

File: admin/class-monk-admin.php [hunk 8]
```diff
@@ -124,45 +124,68 @@ public function monk_add_menu_page() {
...
 			__( 'Default language', 'monk' ),
 			array( $this, 'monk_language_select_render' ),
 			'general_options',
 			'monk_general_options'
 		);
+
+		register_setting( 'general_options', 'monk_active_languages' );
+		add_settings_field(
+			'monk_list',
+			__( 'Add new translation' ),
```
Comment (brenoalvs) [inline admin/class-monk-admin.php]: Missing text-domain

**Review by brenoalvs — CHANGES_REQUESTED** [review]
@chandonn take a look at those changes and pay attention to the code standards.

File: admin/partials/admin-monk-translation-list-render.php (hunk 7 above)
Comment (chandonn-zz) [reply admin/partials/admin-monk-translation-list-render.php]: Fixed

File: admin/class-monk-admin.php (hunk 1 above)
Comment (chandonn-zz) [reply admin/class-monk-admin.php]: Fixed

File: admin/class-monk-admin.php (hunk 8 above)
Comment (chandonn-zz) [reply admin/class-monk-admin.php]: Fixed

File: admin/class-monk-admin.php (hunk 2 above)
Comment (chandonn-zz) [reply admin/class-monk-admin.php]: Fixed

File: admin/class-monk-admin.php (hunk 3 above)
Comment (chandonn-zz) [reply admin/class-monk-admin.php]: Fixed

File: admin/partials/admin-monk-select-render.php (hunk 4 above)
Comment (chandonn-zz) [reply admin/partials/admin-monk-select-render.php]: Fixed

File: admin/partials/admin-monk-translation-list-render.php (hunk 6 above)
Comment (chandonn-zz) [reply admin/partials/admin-monk-translation-list-render.php]: Fixed

File: admin/partials/admin-monk-translation-list-render.php (hunk 5 above)
Comment (chandonn-zz) [reply admin/partials/admin-monk-translation-list-render.php]: Fixed

**Review by brenoalvs — APPROVED** [review]

**Pull request merged**